register_user/register.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
```

Comment:
Move it all to a folder called includes. Create one separate file for doctor_register.php and another for patient_register.php. Move their code to independent files. Then include them where needed. Also move both processes into functions. Make them pure functions, the function should take clear inputs, like username, password, phone number, patient email and return true if the user was created, false if any issues. Redirect accordingly.